webextensions/api/storage.json

```diff
@@ -118,7 +118,7 @@
                 "description": "Supports empty key",
                 "support": {
                   "chrome": {
-                    "version_added": true
+                    "version_added": "≤93"
```

Comment:
@queengooborg the referenced PR that added these were dated Sep 20, 2022. Therefore, shouldn't this be version 105?

Reply:
Uh oh -- 105 is just barely too early to satisfy the linter...

Reply:
@dotproto any idea on how to find out when CHrome started support for empty kets in storage.get?

Reply:
Honestly?  You would have a better idea about how to track that down than I would...

Chrome 105 is _very_ close to being within range, though -- in fact, it's just one version away.  Perhaps we could set this to 104 and call it good?  (The other alternative is to wait until September, once 105 is older than two years?)